libraries/AP_ExternalAHRS/AP_ExternalAHRS_LORD.cpp

```diff
@@ -575,14 +575,14 @@ float AP_ExternalAHRS_LORD::extract_float(const uint8_t *data, uint8_t offset) c
 {
     uint32_t tmp = be32toh_ptr(&data[offset]);
 
-    return *reinterpret_cast<float*>(&tmp);
+    return to_float(tmp);
```

Comment:
a be32tofloat_ptr() method would seem to be ideal

Reply:
Added the new method(s) in 503c3c76ae616be32c2a7b8c408126b4e43857b6. 
Used them here in 06b5ea2fe99ce12dfe31386eeb983bb98af0dd61.